input/script_utils.go

```diff
@@ -64,6 +66,17 @@ type Signature interface {
 	Verify([]byte, *btcec.PublicKey) bool
 }
 
+// ParseSignature parses a raw signature into an input.Signature instance. This
+// routine supports parsing normal ECDSA DER encoded signatures, as well as
+// schnorr signatures.
+func ParseSignature(rawSig []byte) (Signature, error) {
```

Comment:
can we remove this and instead determine that it's a taproot channel and then parse it appropriately?

Reply:
So this is used in `briefcase.go` for the sign details, we may not have enough context there. But we'd only ever write this to disk if it was validated as a proper signature, in which case the small edge case signature wouldn't be feasible to create.

Reply:
I think it is feasible to create if somebody has many computers. I mean we could disallow 64-byte signatures in the non-taproot case. There's also from what I can tell an incentive to try and create a 64-byte DER sig